packages/intuit/src/Utils.js

```diff
@@ -0,0 +1,45 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  logResponse,
+} from '@openfn/language-common/util';
+
+export const prepareNextState = (state, response) => {
+  const { body, ...responseWithoutBody } = response;
+
+  if (!state.references) {
+    state.references = [];
+  }
+
+  return {
+    ...composeNextState(state, response.body),
+    response: responseWithoutBody,
+  };
+};
+
+export const request = (state, method, path, options = {}) => {
+  const { baseUrl, access_token } = state.configuration;
```

Comment:
Can we put some kind of guard on state.configuration just in case it's not defined (likely to happen in local dev)